src/map/Map.js

```diff
@@ -1339,7 +1341,7 @@ export var Map = Evented.extend({
 		for (var i = 0; i < targets.length; i++) {
 			targets[i].fire(type, data, true);
 			if (data.originalEvent._stopped ||
-				(targets[i].options.nonBubblingEvents && Util.indexOf(targets[i].options.nonBubblingEvents, type) !== -1)) { return; }
+				(targets[i].options.bubblingEvents === false && Util.indexOf(this._nonBubblingEvents, type) !== -1)) { return; }

```

Comment:
With my suggestions, this condition would turn into:

```js
targets[i].options.bubblingMouseEvents === false && Util.indexOf(this._mouseEvents, type) !== -1)
```

...which looks clearer at least to me.